web.tf

```diff
@@ -15,7 +15,7 @@ resource "aws_security_group" "acme_web" {
     from_port   = 22
     to_port     = 22
     protocol    = "tcp"
-    cidr_blocks = ["10.0.0.0/24"]
```

Comment:
**Accurics remediation**
 
**[HIGH]: http port open to internet**
 
**Vulnerability**
http port open to internet
 
**Remediation**
Change cidr_block to non 0.0.0.0/0